### ISSUE TITLE
Parse date strings to Date() objects

### ISSUE THREAD

**Issue by baspiotr** [insanoid/SwiftyJSONAccelerator#76]
Is it possible for the soft to detect dates using, for example, regex? If yes, it may be possible to perform the feature that would parse the date in strings into Date() objects.

**Comment by Miranto** [comment]
Yeah, that would be great idea, because we shouldn't keep Dates as Strings in our Model.

**Comment by insanoid** [comment]
Interesting idea, I will add it to the next release list. I am currently not actively updating the project but I will get back to it soon. If this is something you have a solution for I welcome PRs on the project (on the release branch):)

**Comment by nitinalabur** [comment]
@baspiotr @Miranto would NSDataDetector help instead of regex? It should work better than regexing.  more info about nsdatadetectro [here ](http://nshipster.com/nsdatadetector/)
